ovos_gui_plugin_shell_companion/brightness.py

```diff
@@ -232,36 +298,73 @@ def get_sunset_time(self, message=None):
                                                  day=now_time.day, hour=22)
             self.sunrise_time = self.sunset_time + timedelta(hours=8)
 
-        # check sunset times again in 24 hours
+        # check sunset times again in 12 hours
         self.event_scheduler.schedule_event(self.get_sunset_time,
-                                            when=now_time + timedelta(hours=24),
+                                            when=now_time + timedelta(hours=12),
                                             name="ovos-shell.suntimes.check")
```

Comment:
In testing, I keep getting repeated logs about events being in the past so they fire immediately, creating an infinite loop. Seems like it's not getting tomorrow's time, it's getting today's from earlier. I'm testing at night